### PR TITLE
Update disk space needed for Mac install

### PR DESCRIPTION
**Pull request by sfshaza2**
Fixes https://github.com/flutter/website/issues/2995

### CODE REVIEW

File: src/docs/get-started/install/macos.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ To install and run Flutter,
 your development environment must meet these minimum requirements:
 
 - **Operating Systems**: macOS (64-bit)
-- **Disk Space**: 700 MB (does not include disk space for IDE/tools).
+- **Disk Space**: 2.200 MB (does not include disk space for IDE/tools).
```
Comment (tvolkert) [inline src/docs/get-started/install/macos.md]: Is this 2.2 GB?  I calculate 1.98 GB on my local machine.  And the previous version I calculate at 1.51 GB, so I'm curious what the 700 MB is measuring?

Comment (sfshaza2) [reply src/docs/get-started/install/macos.md]: I don't know, @tvolkert. I'm happy to modify it to 2 GB or even 1..98 GB. I'm not sure who performed the calculations, or how it was calculated.

Comment (tvolkert) [reply src/docs/get-started/install/macos.md]: Where'd you see/get the `2.200 MB` number?

**Comment by sfshaza2** [comment]
In the bug report. (They are linked..)

File: src/docs/get-started/install/macos.md (hunk 1 above)
Comment (sfshaza2) [reply src/docs/get-started/install/macos.md]: From the bug report:  https://github.com/flutter/website/issues/2995

Comment (tvolkert) [reply src/docs/get-started/install/macos.md]: I think that info is old (pre-1.9.1).  I'd update it to say `2.8 GB`

Comment (sfshaza2) [reply src/docs/get-started/install/macos.md]: Thanks! Will do.

File: src/docs/get-started/install/macos.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ To install and run Flutter,
 your development environment must meet these minimum requirements:
 
 - **Operating Systems**: macOS (64-bit)
-- **Disk Space**: 2.200 MB (does not include disk space for IDE/tools).
+- **Disk Space**: 2.8 MB (does not include disk space for IDE/tools).
```
Comment (tvolkert) [inline src/docs/get-started/install/macos.md]: GB, not MB

**Pull request merged**